Software_Carbon_Intensity/case_study_template.md

```diff
@@ -0,0 +1,66 @@
+# Case Study Template
+
+*This is a template to use for case studies that are submitted to apply the SCI. *
+
+*Please delete the text in italics and replace it with the appropriate information.*
+
+*For more information on any of the items, the final reference is the [SCI Specification](https://github.com/Green-Software-Foundation/software_carbon_intensity/blob/main/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md)*
+
+*If you find errors, or have further questions, please feel free to engage with us in the [discussions](https://github.com/Green-Software-Foundation/software_carbon_intensity/discussions) section.*
+
+## Overview
+
+_Please provide information describing the use case in a few bullet points_
+
+## Architecture for the system under consideration
+
+_An architecture diagram of the system described in this use case_
+
+### Technical details of the components in the architecture
+
+_Textual description with technical details of each component provided in the architecture diagram_
+
+## Sites for Software Sustainability Actions
+
+_For each of the following sub-sections, indicate **where** and **how** actions can be taken in the following categories_:
+
+### Energy Efficiency 
```

Comment:
In this section are you asking people to write out some ideas for how they might be able to reduce their score?

Reply:
So asking for more details on specific things that they are doing to reduce the score by indicating the "where" and "how" of the actions